cpukit/score/src/threadprioritynode.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Thread Priority Node
+ * 
+ * Copyright (C) 2017. Gedare Bloom and Samman Bikram Thapa.
```

Comment:
Add a new copyright line for yourself, and use the right year, e.g.,
` * Copyright (C) 2018. Samman Bikram Thapa.`